### PR TITLE
Added maven wrapper support

### PR DESCRIPTION
**Pull request by UtkarshDeoli**
I downloaded the repository installed maven and then added maven wrapper to the directory. 
(io.takari:maven:0.7.7:wrapper)

### CODE REVIEW

**Comment by UtkarshDeoli** [comment]
"Codacy Static Code Analysis — 100 new issues of at least minor severity (0 max.)"
can someone help me ?

**Comment by hendrikebbers** [comment]
@UtkarshDeoli the Codacy Plugin is not finally configured. Therefore it currently shows always problems and you can ignore it for now :)
I will do a review of your PR today

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hendrikebbers) [inline .classpath:1]: Please add this file to `.gitignore`

File: .project [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hendrikebbers) [inline .project:1]: Please add this file to `.gitignore`

File: .settings/org.eclipse.jdt.apt.core.prefs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (hendrikebbers) [inline .settings/org.eclipse.jdt.apt.core.prefs:1]: Please add the `.settings` folder to `.gitignore`. All mentioned additions for `.gitIgnore` are your specific eclipse configuration of the project and should not be committed ;)

**Review by hendrikebbers — CHANGES_REQUESTED** [review]
Please add your Eclipse Config files to `.gitignore` :) Omnce this is done I will merge this PR. Very good work @UtkarshDeoli

File: .settings/org.eclipse.jdt.apt.core.prefs (hunk 3 above)
Comment (UtkarshDeoli) [reply .settings/org.eclipse.jdt.apt.core.prefs:1]: hey , i am sorry i am new but , i din't use eclipse , where can i get eclipse config files ?

Comment (UtkarshDeoli) [reply .settings/org.eclipse.jdt.apt.core.prefs:1]: hey , i am sorry i am new but , i din't use eclipse , where can i get eclipse config files ?

Comment (hendrikebbers) [reply .settings/org.eclipse.jdt.apt.core.prefs:1]: no problem :) I will help you and write a more detailed answer

**Comment by hendrikebbers** [comment]
@UtkarshDeoli In general several files (like the `.classpath` file or the `.settings` folder) that are created by your IDEA (Eclipse, Netbeans, Visual Studio Code, IntelliJ - whatever you are using) has been generated on you machine and now are part of this pull request. Since the files are generated as configuration for the IDE on your machine we do not need such files in the global repository. Such files can be ignored from a commit by adding them to the `.gitignore` file. A sample can be found [here](https://gist.github.com/chhh/4961200). You will find much information online when googling for `.gitignore`. I hope this helps. Otherwise please ping me :)

**Pull request closed without merging**